model/src/main/java/com/epam/esm/entity/Order.java

```diff
@@ -0,0 +1,113 @@
+package com.epam.esm.entity;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "order")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "cost")
+    private BigDecimal cost;
+
+    @Column(name = "purchase_date", updatable = false, columnDefinition = "TIMESTAMP() default CURRENT_TIMESTAMP() on update CURRENT_TIMESTAMP()")
+    @CreationTimestamp
+    private ZonedDateTime purchaseDate;
+
+    @ManyToOne
+    @JoinColumn(name = "id_user", referencedColumnName = "id")
+    private User user;
+
+    @ManyToMany
+    @JoinTable(
```

Comment:
If we've update certifcate's price, we will see discrepancy in order cost and certificate price sum. Could we save certificate price at the moment of order creation?

Reply:
Should I make another field in Order class which will save "at the moment" certificate price?